src/niscope/custom_types/measurement_stats.py

```diff
@@ -41,3 +44,7 @@ def __str__(self):
         string_representation += row_format_d.format('Number in Stats', self.num_in_stats)
 
         return string_representation
+
+
+def _populate_channel_and_record_info(measurement_stats, channels, records):
```

Comment:
Why do you need this at all? it is just a straight up wrapper.

Reply:
Because I felt it would be weird to use a helper defined in `waveform_info` module to populate members of `measurement_stats` class. But I don't feel strongly either way. I can get rid of it, if you prefer.

Reply:
Removed the wrapper function in `measurement_stats.py`